def divisao(x: int) -> float:
    return x / 2

def informacoes() -> dict:
    return {
        "is ok": True,
        "name": "Jo√£o Pedro",
        "age": 24
    }

def func_de_erro():
    raise Exception("Algo deu errado")